wabee/tools/base_model.py

```diff
@@ -8,8 +8,9 @@ class ImageToolResponse(BaseModel):
 class StructuredToolResponse(BaseModel):
     variable_name: str = Field(description="An intuitive name for a variable that can be used to easily infer what's stored in it. Use specific names that take the variable content into consideration.")
     content: str = Field(description="The content of the tool response.")
-    local_file_path: Optional[str] = Field(None, description="Optional path to a local file that contains the full content for what is needed.")
-    metadata: Optional[dict] = Field(None, description="Additional metadata to be stored with the response.")
-    memory_push: bool = Field(False, description="Indicates whether tool response should be added to memory")
-    images: Optional[List[ImageToolResponse]] = Field(None, description="Optional list of images that are part of the response.")
-    error: Optional[str] = Field(None, description="Use this field to include an error message if an error occurred during the tool execution.")
+    error: str | None = Field(default=None, description="Use this field to include an error message if an error occurred during the tool execution.")
+
+    local_file_path: str | None = Field(default=None, description="Optional path to a local file that contains the full content for what is needed.")
+    metadata: dict | None = Field(default=None, description="Additional metadata to be stored with the response.")
```

Comment:
Based on the usage of protobuf clients, it seems that Dict and Lists, when not informed, will be used as {} and []. Consider if it may make sense to keep aligned with this behavior in this change.